common/decisions/nato_decisions.txt

```diff
@@ -1,7 +1,8 @@
 ﻿form_nato = {
 	is_shown = {
+		exists = c:USA
+		this = c:USA

```

Comment:
Paradox patched this and so 
`this ?= c:USA` works just fine now 😃

Reply:
That's what was in the decision before, but it caused error spam when Russia/USA was annexed.